teleinfo: update documentation since sensor/text_sensor move. (#871)
OttoWinter/esphomedocs: components/sensor/teleinfo.rst

- **uart_id** (*Optional*, :ref:`config-id`): Manually specify the ID of the :ref:`UART Component <uart>` if you want
  to use multiple UART buses.

> In a teleinfo sensor, in addition to normal sensor configuration: :ref:`Sensor <config-sensor>`

Reviewer: ```suggestion
Sensor
******
```

> - **tag_name** (**Required**): Specify the tag you want to retrieve from the Teleinformation and select with what name to transmit it.

Reviewer: The name to transmit it is the sensor name itself? not the name of the tag_name, right?
```suggestion
- **tag_name** (**Required**, string): Specify the tag you want to retrieve from the Teleinformation.
- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
- All other options from :ref:`Sensor <config-sensor>`.
```

> In a text sensor, in addition to normal text sensor configuration: :ref:`Text Sensor <config-text_sensor>`

Reviewer: ```suggestion
Text Sensor
***********
```

> - **tag_name** (**Required**): Specify the tag you want to retrieve from the Teleinformation and select with what name to transmit it.
> 
> In a text sensor, in addition to normal text sensor configuration: :ref:`Text Sensor <config-text_sensor>`
> 
> - **tag_name** (**Required**): Specify the tag you want to retrieve from the Teleinformation and select with what name to transmit it.

Reviewer: The name to transmit it is the sensor name itself? not the name of the tag_name, right?
```suggestion
- **tag_name** (**Required**): Specify the tag you want to retrieve from the Teleinformation.
```

> - **tag_name** (**Required**): Specify the tag you want to retrieve from the Teleinformation and select with what name to transmit it.

Reviewer: The name to transmit it is the sensor name itself? not the name of the tag_name, right?
```suggestion
- **tag_name** (**Required**, string): Specify the tag you want to retrieve from the Teleinformation.
- **id** (*Optional*, :ref:`config-id`): Manually specify the ID used for code generation.
- All other options from :ref:`Text Sensor <config-text-sensor>`.
```
